src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,38 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  active: boolean;
+  selectedUserId: number | null;
+  onClose: () => void;
+  selectedTodo: Todo | null;
+}
+
+export const TodoModal: React.FC<Props> = ({ active, selectedUserId, onClose, selectedTodo }) => {
+  const [loading, setLoading] = useState(true)
```

Comment:
Consider adding a semicolon at the end of this line for consistency with the rest of the code.